buildSrc/src/test/java/org/opensearch/gradle/precommit/DependencyLicensesTaskTests.java

```diff
@@ -239,28 +243,45 @@ public void givenProjectWithADependencyWithWrongShaThenShouldThrowException() th
 
     @Test
     public void givenProjectWithADependencyMappingThenShouldReturnSilently() throws Exception {
-        project.getDependencies().add("compile", dependency);
+        project.getDependencies().add("compileClasspath", dependency);
 
         File licensesDir = getLicensesDir(project);
-        createAllDefaultDependencyFiles(licensesDir, "groovy");
+        createAllDefaultDependencyFiles(licensesDir, "groovy", "javaparser");
 
         Map<String, String> mappings = new HashMap<>();
-        mappings.put("from", "groovy-all");
-        mappings.put("to", "groovy");
+        mappings.put("from", "javaparser-core");
+        mappings.put("to", "javaparser");
 
         task.get().mapping(mappings);
         task.get().checkDependencies();
     }
 
     @Test
     public void givenProjectWithAIgnoreShaConfigurationAndNoShaFileThenShouldReturnSilently() throws Exception {
-        project.getDependencies().add("compile", dependency);
+        project.getDependencies().add("compileClasspath", dependency);
 
         File licensesDir = getLicensesDir(project);
-        createFileIn(licensesDir, "groovy-all-LICENSE.txt", PERMISSIVE_LICENSE_TEXT);
-        createFileIn(licensesDir, "groovy-all-NOTICE.txt", "");
+        createFileIn(licensesDir, "groovy-LICENSE.txt", PERMISSIVE_LICENSE_TEXT);
```

Comment:
Gradle 7 comes with Groovy 3, the distribution is split into many modules instead of `groovy-all` as before